### ISSUE TITLE
launcher 1.4 not working

### ISSUE THREAD

**Issue by erb1991** [griffenx/Apex-Launcher#10]
have been unable to get launcher to load game properly. have tried uninstalling the old version and reinstalling the new version, but still will not load, have also tried to run the old versions with same lack of success.

**Comment by griffenx** [comment]
I'm going to need details about your issue. Please see [this page](https://github.com/griffenx/Apex-Launcher/blob/master/CONTRIBUTING.md) for issue report requirements.

**Comment by erb1991** [comment]
I have launcher 1.4 I believe. I load the launcher up and everything works fine until I hit start game. It tells me it'd unable to find my game.exe file and ask me if I wish to download it. I chose yes, it downloads to my download folder, I then go and extract the files and place them in a new folder. I then go back into the laucher, click the settings and make sure I have it linked to the new folder of the extracted game files and save. I then try to load the game again  but still receive the unable to locate game.exe  message

**Comment by griffenx** [comment]
I see your problem. You're not supposed to go messing with the files. The whole point of the launcher is that it handles the downloading and extracting of the files for you.

The way the install path works is it creates a subfolder called Versions in whatever directory you've pointed it to and downloads and extracts  each game version in its own folder there. It then looks for that specific directory tree when trying to launch the game. 

TL;DR Don't mess with the files. Let the launcher do its job and you should be fine.

**Comment by griffenx** [comment]
I'm going to assume that this worked, so I'll close the issue.

**Comment by erb1991** [comment]
I'm not sure if it worked because my laptop black screened on me and died
but I did get it to work on the desktop
